source/manual/alerts/search-spelling-suggestions.html.md

```diff
@@ -0,0 +1,23 @@
+---
+owner_slack: '#search-team'
+review_by: 2017-06-26
+title: 'Test spelling suggestions work for example queries failed'
```

Comment:
I had to read this a couple of times before I understood what was going on :)

Would something like "Check for spelling suggestions failed" work?